README.md

```diff
@@ -28,6 +28,15 @@ These modifications are then re-pushed to the corresponding major tags (e.g. `v4
      key: node_modules
 ```
 
+## Running with your own implementation of GitHub Actions Cache API
+
+If you have implemented the GitHub Actions Cache API alongside self-hosted runners, you can enable its use by setting the `CIRRUS_HTTP_CACHE_HOST` environment variable when starting your runner:
+
+```bash
+export CIRRUS_HTTP_CACHE_HOST=cache.internal:8080
+./run.sh
```

Comment:
It's not clear what is `run.sh` and why would anyone want to run it this way?

I've thought that simply explaining the difference from the `actions/cache` action should be more than enough.

Reply:
`run.sh` is part of Actions Runner installation that people should be aware of. IMO right now one really needs to think that this action can be used with not Cirrus Runners. This is why I want to explicitly document it.